<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bsa</class>
 <widget class="QDialog" name="bsa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BSA</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>BSA</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BSA wynosi [m^2]:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_oblicz">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Oblicz</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Waga [kg]:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_waga">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wzrost [cm]:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_wroc">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wróć</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_wzrost">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_bsa">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" alignment="Qt::AlignVCenter">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="plainText">
        <string>Wyniki wyszukiwania
Fragment z odpowiedzią z internetu
BSA – powierzchnia ciała. Parametr BSA jest niezbędny do określenia dawkowania wielu leków oraz obliczenia niektórych parametrów fizjologicznych (np. wskaźnika sercowego). Do obliczenia konieczna jest masa ciała i wzrost pacjenta.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
